pkg/testing/tools/tools.go

```diff
@@ -36,6 +36,22 @@ func IsPolicyRevision(ctx context.Context, t *testing.T, client *kibana.Client,
 	}
 }
 
+func GetUninstallToken(ctx context.Context, kibClient *kibana.Client, policyId string) (string, error) {
+	resp, err := kibClient.GetPolicyUninstallTokens(ctx, policyId)
+	if err != nil {
+		return "", fmt.Errorf("failed to fetch uninstall tokens: %w", err)
+	}
+	if len(resp.Items) == 0 {
+		return "", fmt.Errorf("expected non-zero number of tokens: %w", err)
```

Comment:
In this instance `err == nil` so we shouldn't wrap a nil error 
```suggestion
		return "", fmt.Errorf("expected non-zero number of tokens")
```